lib/index.js

```diff
@@ -326,6 +326,16 @@ export default {
 						// 'appc:closeRelated': () => related.closeRelatedFiles({forceAllClose: true})
 					})
 				);
+
+				// add contect menu
+				atom.contextMenu.add({
+					'atom-workspace': [
+						{label: "Open related view", command:'appc:open-related-view', shouldDisplay: () => this.checkShouldDisplay("xml")},
+						{label: "Open related style", command:'appc:open-related-style', shouldDisplay: () => this.checkShouldDisplay("tss")},
+						{label: "Open related controller", command:'appc:open-related-controller', shouldDisplay: () => this.checkShouldDisplay("js")},
```

Comment:
<!--
  5 failure:  lib/index.js line..., lib/index.js line..., lib/index.js line..., lib/index.js line..., lib/index.js line...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: lib/index.js;
  Line: 335;
-->
- :no_entry_sign: lib/index.js line 335 – A space is required after '{'. (object-curly-spacing)
- :no_entry_sign: lib/index.js line 335 – A space is required before '}'. (object-curly-spacing)
- :no_entry_sign: lib/index.js line 335 – Missing space before value for key 'command'. (key-spacing)
- :no_entry_sign: lib/index.js line 335 – Strings must use singlequote. (quotes)
- :no_entry_sign: lib/index.js line 335 – Strings must use singlequote. (quotes)